Src/BootCamp.Chapter/Car.cs

```diff
@@ -1,14 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace BootCamp.Chapter
+﻿namespace BootCamp.Chapter
 {
     // This should have the following attribute
-    // [Textable]
+    [Textable]
     public class Car
     {
+        [Textable]
```

Comment:
You don't format individual properties in their own boxes, just the whole car.

Reply:
hmm, I did but then the attribute did not work at all

Reply:
then this line gives null :  `requiredAttributeTextable = AttributesGetter.GetPropertyAttribute<Textable>(prop); `